File: Python/BinaryTree/BinaryTree.py
# a python class that represents an individual node in a binary tree

class Node:
    def __init__(self, key):
        self.left = None
        self.right = None
        self.value = key


# create root 
root = Node(1)
''' following is the tree after above statement
        1
    / \
    None None'''

root.left = Node(2)
''' 2 and 3 become left and right children of 1
        1
        / \
        2     3
    / \ / \
None None None None'''

root.right = Node(3)

File: Javascript/BinaryTree/BinaryTree.js
// Binary tree using javascript

class Node {
    constructor(data){
        this.data = data
        this.left = null
        this.right = null
    }
}

function inorderTraversal(root) {

    if (root == null) {
        return
    }

    inorderTraversal(root.left)

    console.log(root.data);
    
    inorderTraversal(root.right)
}

function postorderTraversal(root) {
    
    if (root == null) {
        return
    }

    postorderTraversal(root.left)

    postorderTraversal(root.right)

    console.log(root.data);
}

root = new Node(1)
root.left = new Node(2)
root.right = new Node(3)
root.left.left = new Node(4)
root.left.right = new Node(5)

console.log("Inorder Traversal");
inorderTraversal(root)

console.log("Postorder Traversal");
postorderTraversal(root)